spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/TaskConfiguration.java

```diff
@@ -130,55 +128,6 @@ public TaskPlatform localTaskPlatform(LocalPlatformProperties localPlatformPrope
 		return taskPlatform;
 	}
 
-	/**
-	 * The default profile is active when no other profiles are active. This is configured so
-	 * that several tests will pass without having to explicitly enable the local profile.
-	 * @return a no-op {@link SchedulerService}
-	 */
-	@Profile({ "local", "default" })
-	@Bean
-	public SchedulerService schedulerService() {
```

Comment:
It wasn't just for tests. We did this dump thing for UI not to throw errors too early as essentially we wanted to fail only when schedule we tried to use. Put it back please.

Reply:
ok, Though this bean might be needed to fail fast, the availability of `SchedulerService` by default or in the `local` setup looks incorrect. Could you tell me which part of the UI requires this bean? I will try to see if we can work around it by not having this bean explicitly?

Reply:
Hmm hold on, I was expecting to get an error before even appempting to create a schedule. Think I was wrong with my initial comment.

Reply:
We added some of these dump empty impls for local not to fail until attempting to actually schedule a task. It was a convenience for developer and not to throw 505's in UI. All these seem to work as is with this pr.